### DISCUSSION TITLE
Cannot serve local files in MLFLOW server UI because of Sandboxing.

### DISCUSSION THREAD

**Discussion by iamishaan** [mlflow/mlflow#7889] [Q&A]
Hey Team, I am facing the following issue in the ML UI . I have an iFrame element that has a sandbox attribute that doesn't let me load/serve local files in the mlflow server UI.  Removing the sandbox attrib from the console fixes the issue and lets me load the files. I have tried adding headers in my nginx config to allow cross origin and X-Frame headers as well. Is there any way I can fix it externally. Attaching the screenshots of the error and the iFrame element. Thanks folks.
<img width="905" alt="Screenshot 2023-02-21 at 2 04 39 PM" src="https://user-images.githubusercontent.com/18017901/221093348-c7c9e7bd-700d-472b-bd58-c5179e8aeac0.png">

**Comment by JHibbard** [comment]
Hi iamishaan,

Could you provide more info on your deployment scenario, a screenshot of you attempting to upload the file so that I can see the UI element, and share your MLflow version? It'd help with troubleshooting.

Thank you!
Jim
